dotnet/src/SemanticKernel.Core/TemplateEngine/Blocks/NamedArgBlock.cs

```diff
@@ -61,6 +56,22 @@ public NamedArgBlock(string? text, ILoggerFactory? logger = null)
         }
     }
 
+    /// <summary>
+    /// Attempts to extract the name and value of a named argument block from a string
+    /// </summary>
+    /// <param name="text">String from which to extract a name and value</param>
+    /// <param name="argParts">This array has its first and second elements set to the extracted name and value when successful.
+    /// An empty array is returned otherwise.</param>
+    /// <returns>true when a name and value are successfully extracted from the given text</returns>
+    internal static bool TryGetNameAndValue(string? text, out string[] argParts)
```

Comment:
Consider defining this as `bool TryGetNameAndValue(string? text, out string name, out string value)`

Reply:
I like it